### PR TITLE
Adds Discord Notification Documentation

### PR DESCRIPTION
**Pull request by night**
This is a PR for documentation for https://github.com/travis-ci/travis-tasks/pull/71

### CODE REVIEW

File: user/notifications.md [hunk 1]
```diff
@@ -325,6 +325,115 @@ Other flags, such as `on_success` and `on_failure` also work like they do in IRC
 
 Pull Request builds do not trigger Campfire notifications.
 
+## Configuring discord notifications
```
Comment (Stanzilla) [inline user/notifications.md]: d => D

File: user/notifications.md [hunk 2]
```diff
@@ -325,6 +325,115 @@ Other flags, such as `on_success` and `on_failure` also work like they do in IRC
...
+
+Travis CI can send build notifications to channels on your [Discord](https://discordapp.com) server.
+
+On Discord, head to a channel's settings and navigate to the webhooks tab.
+
+<figure>
+  <img alt="Screenshot of Discord channel webhooks" src="https://i.imgur.com/mtZOWN8.png"/>
+</figure>
+
+Now create a new webhook using the "Create Webhook" and customize the name. Setting an avatar is not necessary because Travis CI supplies one.
```
Comment (Stanzilla) [inline user/notifications.md]: *button

File: user/notifications.md [hunk 3]
```diff
@@ -325,6 +325,115 @@ Other flags, such as `on_success` and `on_failure` also work like they do in IRC
...
+Now take the webhook ID and the webhook token out of the generated URL. The format of the URL is `https://discordapp.com/api/webhooks/[id]/[token]`
+
+The simplest configuration requires the webhook ID and token you just generated:
+
+```yaml
+notifications:
+  discord: '[id]:[token]'
+```
+
+To specify a multiple channels (even across servers) use:
```
Comment (Stanzilla) [inline user/notifications.md]: -a

File: user/notifications.md [hunk 4]
```diff
@@ -325,6 +325,115 @@ Other flags, such as `on_success` and `on_failure` also work like they do in IRC
...
+> This will look something like the following in your config:
+>
+> ```yaml
+> notifications:
+>   discord:
+>     channels:
+>       - secure: "0vlxmbtg45tnayuf836eegt158hrmjbg0fu6rg56zpn9k0bcw6ra8abvkasn="
+> ```
+
+### Customizing discord notifications
```
Comment (Stanzilla) [inline user/notifications.md]: d => D

File: user/notifications.md (hunk 1 above)
Comment (night) [inline user/notifications.md]: this is intentional to match the other headings

File: user/notifications.md (hunk 4 above)
Comment (night) [inline user/notifications.md]: this is intentional to match the other headings

File: user/notifications.md (hunk 3 above)
Comment (AraHaan) [inline user/notifications.md]: ``To specify multiple channels (even across servers) use:``

**Comment by StarSmasher44** [comment]
Come one merge this and the other PR, we need this

**Comment by BanzaiMan** [comment]
As indicated in https://github.com/travis-ci/travis-tasks/pull/71#issuecomment-294579129, Discord support is on hold.

**Comment by StarSmasher44** [comment]
Yeah, but why?

**Comment by TobiTenno** [comment]
So, why is it on hold? I don't think y'all are gaining anything and are in fact losing respect when you leave things like this open for several months with no reason why they're held back.

**Comment by paulsapps** [comment]
Isn't the feature merged now.. surely needs the docs now too? :joy:

**Comment by paulsapps** [comment]
Oh sorry, I saw approved and assumed it was merged, ha :).

**Comment by plaindocs** [comment]
Hey folks, thanks for your work on this. I'm closing for now, happy to reopen when https://github.com/travis-ci/travis-tasks/pull/71 becomes active.

**Pull request closed without merging**